students/ThanhNhan/lesson02/assignment2/print_grid2.py

```diff
@@ -0,0 +1,21 @@
+def print_grid2(length, minus): 
+    plus = "+"   
+    minus_num = minus * "-" 
```

Comment:
careful with variable names here -- I would expect a name like "minus_num" to be a number.